src/api/js/z3.cc.j2

```diff
@@ -0,0 +1,161 @@
+/* Automatically generated file */
```

Comment:
perhaps this file is _not_ automatically generated.

Reply:
Fair point. It does make it seem as though it marks the `.j2` file as autogenerated, in addition to injecting the line in the actual generated file.